doc/guides/tfm/build.rst

```diff
@@ -23,7 +23,7 @@ Images Created by the TF-M Build
 The TF-M build system creates the following executable files:
 
 * tfm_s - TF-M secure firmware
-* tfm_ns - a nonsecure app which is discarded in favor of the Zephyr app
+* tfm_ns - TF-M non-secure app (only used by regression tests).
```

Comment:
Thanks for pointing this out.

If we only use this for integration tests, wouldn't it be good to introduce a `CONFIG_TFM_NS` to control build of the TF-M ns app, similar to `CONFIG_TFM_BL2` ?
Then integration tests can enable building of TF-M ns when needed.

I see no reason for building the TF-M ns everytime if that is not needed.

If agreeing with this, please create an enhancement task for this work, as I believe  such cleanup to be outside scope of this PR.

Reply:
Yes, that is a good point. I'll create an internal task for this.

There is a different Kconfig that determines which application is flashed, but it is tied to the regression tests in naming.

Reply:
Follow up: https://github.com/zephyrproject-rtos/zephyr/pull/43599